README.md

```diff
@@ -6,8 +8,6 @@
 
 **⚠️ This project is WIP and not ready for production use yet!**
 
-Currently we link to the latest build of [Primer CSS] so that we may use current Primer styles to start to build components. This does not include `primer-base` so as to avoid unwanted base overrides.
```

Comment:
Didn't feel like this was necessary to include anymore since we've got the section on BaseStyles and our CSS is injected